js/common/modules/@arangodb/graph/pregel-test-helpers.js

```diff
@@ -1186,10 +1284,160 @@ function makeSeededPagerankTestSuite(isSmart, smartAttribute, numberOfShards) {
     };
 }
 
+function makeSSSPTestSuite(isSmart, smartAttribute, numberOfShards) {
+
+    const verticesEdgesGenerator = loadGraphGenerators(isSmart).verticesEdgesGenerator;
+    // const makeEdgeBetweenVertices = loadGraphGenerators(isSmart).makeEdgeBetweenVertices;
+
+
+
+    // only for debugging
+    const printVertexKey = function (vertex) {
+        console.warn(`printVertexKey: ${vertex._key}`);
+    };
```

Comment:
if this is only for debugging, remove it or move it into some higher-level utility file.